### ISSUE TITLE
Task :react-native-cardview:verifyReleaseResources FAILED

### ISSUE THREAD

**Issue by mathnogueira** [Kishanjvaghela/react-native-cardview#40]
EDIT: seems that the PR #39 already fixed the problem, however the fix wasn't published to the npm registry yet.

**Describe the bug**
When building the app using `compileSdkVersion` as 28, the build fails with the following output:

```
> Task :react-native-cardview:verifyReleaseResources FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':react-native-cardview:verifyReleaseResources'.
> java.util.concurrent.ExecutionException: com.android.builder.internal.aapt.v2.Aapt2Exception: Android resource linking failed
  /home/matheus/Projects/patrulha-2.0/node_modules/react-native-cardview/android/build/intermediates/res/merged/release/values-v28/values-v28.xml:7: error: resource android:attr/dialogCornerRadius not found.
  /home/matheus/Projects/patrulha-2.0/node_modules/react-native-cardview/android/build/intermediates/res/merged/release/values-v28/values-v28.xml:11: error: resource android:attr/dialogCornerRadius not found.
  /home/matheus/Projects/patrulha-2.0/node_modules/react-native-cardview/android/build/intermediates/res/merged/release/values/values.xml:2737: error: resource android:attr/fontVariationSettings not found.
  /home/matheus/Projects/patrulha-2.0/node_modules/react-native-cardview/android/build/intermediates/res/merged/release/values/values.xml:2738: error: resource android:attr/ttcIndex not found.
  error: failed linking references.


* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org
```

The problem seems to be that `react-native-cardview` sets `compileSdkVersion` as 27 in `android/build.gradle`. Changing it to 28 solves the problem.

**Comment by liganghui** [comment]
I have the same problem.
Please update as soon as possible :broken_heart:

**Comment by chrisella** [comment]
Same problem for me.

**Comment by mathnogueira** [comment]
A temporary fix would be using:
```
"react-native-cardview": "https://github.com/Kishanjvaghela/react-native-cardview.git#f6905ac21003117e6154faa5da14bdd2fdad904c"
```
in your `package.json`

**Comment by Kishanjvaghela** [comment]
Fixed c9d60719311c72f6c09b66c5966646834bf6a6fc
